help.go

```diff
@@ -115,18 +116,18 @@ func printApp(w *helpWriter, app *Application) {
 	}
 }
 
-func printCommand(w *helpWriter, app *Application, cmd *Command) {
+func printCommand(w *helpWriter, app *Application, cmd *Command, groups map[string]Group) {
```

Comment:
Per other comment, the groups should be accessible through the model, in which case this should be unnecessary.

Reply:
Same thing for all the other locations.